asterix/COPYING

```diff
@@ -0,0 +1,674 @@
+GNU GENERAL PUBLIC LICENSE
```

Comment:
I guess we shouldn't put a licence into this dir, instead we should to submodule https://github.com/IQAndreas/markdown-licenses to the root.

Reply:
Will try, don't know how exactly to do that. If you could give me a help it is Wellcome.